new/t2.rb

```diff
@@ -0,0 +1,10 @@
+class Plumbus
+private
```

Comment:
Indent access modifiers like `private`.

```suggestion
  private
```